### PR TITLE
[8.0] Imp partner survey fill

### PR DESCRIPTION
**Pull request by angelmoya**
Allow system users to fill survey as a partner, from user form click on "More/Fill Survey".
On survey finish if your user is not partner of this survey a new link is showed to return to partner form.

### CODE REVIEW

**Comment by angelmoya** [comment]
@rafaelbn , now it finished, and coverage improved, could you change label, or can I do it?

**Comment by angelmoya** [comment]
@rafaelbn could you change labels, it's finished and ready to review

File: partner_survey/wizard/fill_survey.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api, fields
```
Comment (susport) [inline partner_survey/wizard/fill_survey.py]: Order by name according to the guide OCA

**Comment by rafaelbn** [comment]
Rebuilding runbot for testing

**Review by ghost — COMMENTED** [review]
Tested in Runbot, working fine

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**